opencga-core/src/main/java/org/opencb/opencga/core/models/externalTool/ContainerMixin.java

```diff
@@ -0,0 +1,7 @@
+package org.opencb.opencga.core.models.externalTool;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties({"user", "password"})
+public interface ContainerMixin {
```

Comment:
I'd rename this to "ContainerRedactedMixin" or  "ContainerShadowMixin" to make clear what is the purpose of this mixin.